tests/misc_env/exec.py

```diff
@@ -134,6 +134,28 @@ def translate_to_daemon_id(node, string: str) -> str:
     return replaced_string
 
 
+def get_systemctl_service_name(node, service_name, command):
+    """
+    Return the systemctl service name matching the service_name parameter given.
+
+    Args:
+        node:       System to be used to gather the information.
+        service_name:     The string pattern to be used to match the service name.
+        command:        Command to be executed.
+
+    Return:
+        command after adding the service name.
+    """
+    cmd = "touch grepout"
+    out, _ = exec_command(node, sudo=True, cmd=cmd)
+    cmd = "systemctl -l | grep ceph > /root/grepout"
+    out, _ = exec_command(node, sudo=True, cmd=cmd)
+    cmd = f"cat /root/grepout | grep {service_name}"
+    out, err = exec_command(node, sudo=True, cmd=cmd)
+    service_full_name = out.split()[0]
+    return f"{command} {service_full_name}"
```

Comment:
I believe we can do this with a single call
```
# systemctl list-units --type=service | grep ceph | grep alertmanager | cut -d " " -f 1

cmd = f"systemctl list-units --type=service | grep ceph | grep {service_name} | cut -d ' ' -f 1"
out, err = exec_command(node, cmd)
return out.split()[0]
```

Reply:
Changes done - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4GE5AP/